requirements.txt

```diff
@@ -1,4 +1,4 @@
-scipy>=0.11
+scipy>=1.8.0
```

Comment:
```suggestion
scipy>=1.7.0
```

We have 1.7.4 in the CI and it works as expected.

Reply:
Scipy bumped from v1.7.3 to v1.8.0 without having intermediate releases. So I think `scipy>=1.8.0` and `scipy>=1.7.4` are equivalent and I do not think v1.7.3 or v1.7.1 works.

Reply:
We have 1.7.3 (typo above, sorry) here https://github.com/pysal/pointpats/actions/runs/4320355325/jobs/7540519213 and it works apparently. So 1.7 seems to be enough now I believe.

Reply:
This is testing on scipy v1.10.1?
<img width="439" alt="image" src="https://user-images.githubusercontent.com/7359284/222773631-505eb028-ebbe-4f29-8aaf-01765cf99714.png">

Reply:
Ah! I missed this. And It also means that I have not done #87 properly. The minimal env should use the pinned versions of SciPy. Not sure why it gets upgraded by pip...

Reply:
Nope, I am stupid. It is the change in this PR that is forcing the update. 

If 1.8 is the new minimal, can you also update it in the 38-minimal environment? So we really get 1.8 and not 1.10 when we want to test the oldest supported version?

Reply:
Just updated the 38-minimal environment